makefile

```diff
@@ -13,6 +13,8 @@ endif
 endif
 endif
 
+PACKAGE_VERSION :=$(shell cat package.json | jq -r .version)
```

Comment:
This works, but it's a different approach to our notifiers and so I think it might trip us up if we're not careful. (Also, the source of truth being inside the package.json is a bit odd for primarily a Go package.)

Can you take a look at the `Makefile` and `bump-version.sh` script in bugsnag/bugsnag-android-performance? I think that would be our best pattern to follow - the version is either derived from the branch name or entered manually and it sets the values in all the places in the repo that it needs to, including the changelog.

In this case you're using the version in the `Makefile` itself, so we would want to refactor so that it either sets the value in `main.go` source or create a file that just stores the version (e.g. https://github.com/bugsnag/bugsnag-ruby/blob/master/VERSION)